src/inverted-index.js

```diff
@@ -1,85 +1,89 @@
-                uniqueWords = new Set(keywords);
-                for (let key of uniqueWords.keys()) {
-                    if (clean.includes(key)){
-                        if (!wordOccurrence.hasOwnProperty(key)){
-                            wordOccurrence[key] = [];
-                        }
-                        
-                        wordOccurrence[key].push(i);
-                    }
-                }
-            }
-        }
-        return wordOccurrence;
+    return fileIndex;
+  }
+
+  /**
+   * Create Index method that indexes text content of uploaded file
+   * @param {Array} file
+   * @return {Object} wordOccurrence
+   */
+  createIndex(file) {
+    const wordOccurrence = {};
+    const merge = [];
+    let keywords = [];
+    let uniqueWords;
+    if (file.length === 0) {
+      return false;
     }
-    /**
-     * Search Index Method that searches uploaded file with the aid of the created index
-     * @param terms(words to be searched), reff
-     * @return result
-     */
-    searchIndex(terms,reff) {
-        let result = {};
-        if (typeof terms !== 'string' || terms === '' || (terms.trim()) === '') {
-            return false;
-        }
-        else {
-            let token = tokenize(terms);
-            let words = token.split(' ');
-            for (let i = 0; i < words.length; i++) {
-                if (reff[words[i]] === undefined) {
-                    console.log(words[i] + ' : ' + 'Not Found!!!!');
-                }
-                else {
-                    let key = words[i];
-                    let value = reff[words[i]];
-                    result[key] = value;
-                }
-            }
+    for (let count = 0; count < file.length; count += 1) {
+      const clean = tokenize(file[count].text);
+      merge.push(clean.split(' '));
+      keywords = [].concat(...merge);
+      uniqueWords = new Set(keywords);
+      uniqueWords.forEach((key) => {
+        if (clean.includes(key)) {
+          if (!{}.hasOwnProperty.call(wordOccurrence, key)) {
+            wordOccurrence[key] = [];
+          }
+
+          wordOccurrence[key].push(count);
         }
-        return result;
+      });
     }
 
+    return wordOccurrence;
+  }
+
+  /**
+   * Search Index Method that searches uploaded file with
+   * the aid of the created index
+   * @param {String} terms
+   * @param {Array} occurrence
+   * @return {Object} result
+   */
+  searchIndex(terms, occurrence) {
+    const result = {};
+    if (typeof terms !== 'string' || terms === '' || (terms.trim()) === '') {
+      return false;
+    }
+    const token = tokenize(terms);
+    const words = token.split(' ');
+    for (let count = 0; count < words.length; count += 1) {
+      if (occurrence[words[count]] === undefined) {
+        result[words[count]] = 'Not Found';
+      } else {
+        const key = words[count];
+        const value = occurrence[words[count]];
+        result[key] = value;
```

Comment:
The use of variable `value` is redundant here as you only initialized it in the line above just to pass the value to this line.